lftpd.c

```diff
@@ -22,6 +22,9 @@
 #include "private/lftpd_string.h"
 #include "private/lftpd_io.h"
 
+#define VALID_USERNAME "admin"
```

Comment:
Use these from NVS. Same user name and  password as the web login. Use API from nvs manager component to get these values